### PR TITLE
Display workshops on lecturer profile page

### LINKED ISSUE #65: Information about past (and current) lectures on profile page

### PR DESCRIPTION
**Pull request by krzys-h**
This makes lecturer's past and current workshops displayed both in the public profile page, and the "past participation" list

Closes #65

### CODE REVIEW

File: wwwapp/views.py [hunk 1]
```diff
@@ -84,13 +84,26 @@ def profile_view(request, user_id):
...
     if can_see_all_users or is_my_profile:
         context['profile'] = profile
+        context['participation_data'] = profile.all_participation_data()
+        if not can_see_all_workshops and not is_my_profile:
+            # If the current user can't see non-public workshops, remove them from the list
+            for participation in context['participation_data']:
+                participation['workshops'] = [w for w in participation['workshops'] if w.is_publicly_visible()]
+
+    if can_see_all_workshops or is_my_profile:
+        context['lecturer_workshops'] = profile.lecturer_workshops.all()
```
Comment (cytadela8) [inline wwwapp/views.py]: Add ordering here

File: templates/profile.html [hunk 2]
```diff
@@ -145,7 +151,36 @@ <h1>{{ title }}</h1>
               Edytuj
             </a>
           {% endif %}
+        {% endif %}
 
+        {% if lecturer_workshops %}
+          <h3>Prowadził{% if gender == 'F' %}a{% endif %} warsztaty:</h3>
```
Comment (cytadela8) [inline templates/profile.html]: Nice

File: wwwapp/views.py (hunk 1 above)
Comment (krzys-h) [reply wwwapp/views.py]: Done

File: templates/profile.html (hunk 2 above)
Comment (grzegorz225) [inline templates/profile.html]: What about a custom filter?

**Pull request merged**